CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+fixes/changes/improvements via pull requests.
+
+## Submitting issues
+
+Both bug reports and feature requests are welcome.
+Submit issues [here](https://github.com/Kotlin/dataframe/issues).
+
+* Search for existing issues to avoid reporting duplicates.
+* When submitting a bug report:
+    * Test it against the most recently released version. It might have been already fixed.
+    * Include the code that reproduces the problem or attach the link to the repository with the project which fully reproduces the problem.
+    * However, don't put off reporting any weird or rarely appearing issues just because you cannot consistently
+      reproduce them.
+    * If the bug is in behavior, then explain what behavior you've expected and what you've got.
+* When submitting a feature request:
+    * Explain why you need the feature &mdash; what's your use-case, what's your domain.
+    * Explaining the problem you face is more important than suggesting a solution.
+      Report your issue even if you don't have any proposed solution.
+    * If there is an alternative way to do what you need, then show the code of the alternative.
+
+## Submitting PRs
+
+We love PRs. Submit PRs [here](https://github.com/Kotlin/dataframe/pulls).
+However, please keep in mind that maintainers will have to support the resulting code of the project,
+so do familiarize yourself with the following guidelines.
+
+* All development (both new features and bug fixes) is performed in the `master` branch.
+    * Base PRs against the `master` branch.
+    * PR should be linked with the issue,
+      excluding minor documentation changes, the addition of unit tests, and fixing typos.
+* If you make any code changes:
+    * Follow the [Kotlin Coding Conventions](https://kotlinlang.org/docs/reference/coding-conventions.html).
+    * [Build the project](#building) to make sure it all works and passes the tests.
+* If you fix a bug:
+    * Write the test the reproduces the bug.
+    * Fixes without tests are accepted only in exceptional circumstances if it can be shown that writing the
+      corresponding test is too hard or otherwise impractical.
+* If you introduce any new public APIs:
+    * All new APIs must come with documentation and tests.
+    * If you plan API additions, then please start by submitting an issue with the proposed API design
+      to gather community feedback.
+    * [Contact the maintainers](#contacting-maintainers) to coordinate any big piece of work in advance via submitting an issue.
+* If you fix documentation:
+    * If you plan extensive rewrites/additions to the docs, then please [contact the maintainers](#contacting-maintainers)
+      to coordinate the work in advance.
+    * Also, we have a special simple [guide](https://github.com/Kotlin/dataframe/blob/master/docs/contributions.md) how to contribute in the documentation.
+
+## PR workflow
+
+0. Contributor build the library locally and run all unit test via Gradle task `dataframe:test`
```

Comment:
The contributor builds the library locally and runs all unit tests via the Gradle task `dataframe:test`.